app/src/main/java/edu/calvin/cs262/cs262d/eventconnect/views/AddEvent.java

```diff
@@ -53,35 +104,85 @@ public boolean onOptionsItemSelected(MenuItem item) {
         return super.onOptionsItemSelected(item);
     }
 
+    /**
+     *
+     * @param view
+     */
     public void onCreateEventClicked(View view) {
         //access the data from the UI elements
         //TODO: Enforce a title and description??
         String title = eventTitle.getText().toString();
         String desc = eventDescription.getText().toString();
-        String host = eventHost.getText().toString();
         String loc = eventLocation.getText().toString();
```

Comment:
Since you set loc below, you don't need to set it here. (You may need to still declare it here, though).

Reply:
I'm sorry, can you clarify this one? Not sure what you mean by since is et it below, where?